pkg/clusters/addons/kong/kong.go

```diff
@@ -29,14 +30,34 @@ const (
 	AddonName clusters.AddonName = "kong"
 
 	httpPort = 80
+
+	// DefaultEnterpriseImageRepo default kong enterprise image
+	DefaultEnterpriseImageRepo = "kong/kong-gateway"
+
+	// DefaultEnterpriseImageTag latest kong enterprise image tag
+	DefaultEnterpriseImageTag = "2.5.0.0-alpine"
+
+	// KongEnterpriseTestLicense is the kong license data secret name
+	KongEnterpriseTestLicense = "kong-enterprise-license"
+
+	// EnterpriseAdminPasswordSecretName is the kong admin seed password
+	EnterpriseAdminPasswordSecretName = "kong-enterprise-superuser-password"
+
+	// EnterpriseKongAdminDefaultPWD is kong admin password
+	EnterpriseKongAdminDefaultPWD = "password"
 )
 
 // Addon is a Kong Proxy addon which can be deployed on a clusters.Cluster.
 type Addon struct {
-	namespace  string
-	deployArgs []string
-	dbmode     DBMode
-	proxyOnly  bool
+	namespace         string
+	deployArgs        []string
+	dbmode            DBMode
+	proxyOnly         bool
+	enterprise        bool
+	repo              string
+	tag               string
+	license           string
+	kongAdminPassword string
```

Comment:
Similar to comments about the `Builder` attribute names, we should improve the clarity of the attributes in the `Addon` as well.

```suggestion
	enterprise                                 bool
	proxyImage                              string
	proxyImageTag                         string
	enterpriseLicenseSecretName string
	enterpriseSuperuserPassword string
```

Reply:
Addon addressed https://github.com/Kong/kubernetes-testing-framework/pull/104/files#diff-7cc1dbd395c3c0ea6e62a7d5a3d4493f37054d7b7be23f198340cdd6a52c0be2R89